jenkins-pipelines/manager/centos-manager-backup-gce.jenkinsfile

```diff
@@ -7,6 +7,8 @@ managerPipeline(
     backend: 'gce',
     test_name: 'mgmt_cli_test.MgmtCliTest.test_backup_feature',
     test_config: 'test-cases/manager/manager-regression-gce.yaml',
+    scylla_version: '',  // In the manager scylla_version has a value by default,  but hydra currently can't find gce images by specific images.
+    // Therefore, I set scylla_version to be empty for this run, and instead set chose a set a predetermined scylla machine image
```

Comment:
this comment seems a bit confusing (and confused):
`find gce images by specific images` ?

`set chose a set a predetermined scylla machine image` ?